packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -352,6 +364,12 @@ export default {
                 this.$emit(EventNames.CheckoutSetupGuestSuccess);
             } catch (thrownErrors) {
                 this.$emit(EventNames.CheckoutSetupGuestFailure, thrownErrors);
+
+                this.$logger.logError(
+                    'Checkout Setup Guest Failure',
+                    this.$store,
```

Comment:
How comes we're emitting the `store` as part of the error?

Reply:
@kevinrodrigues This is so we can get all the data we need in CoreWeb.

Reply:
Right i see. 👍